main.py

```diff
@@ -1,10 +1,20 @@
-import cv2
+import cv2, face_recognition
 #get webcam
 cap = cv2.VideoCapture(0)
```

Comment:
Using zero here is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=Magic%20numbers%20are%20common%20in%20at%20program%20run-time.).

Use a constant at the top of the file, assign 0 to it, then use the constant value here.
This also futureproofs the code against any necessary changes in the future, such as selecting a camera index if there are several.